.yamllint

```diff
@@ -0,0 +1,48 @@
+---
+rules:
+  braces:
+    min-spaces-inside: 1
+    max-spaces-inside: 1
+    min-spaces-inside-empty: 0
+    max-spaces-inside-empty: 0
+  brackets:
+    min-spaces-inside: 0
+    max-spaces-inside: 0
+    min-spaces-inside-empty: 0
+    max-spaces-inside-empty: 0
+  colons:
+    max-spaces-before: 0
+    max-spaces-after: 1
+  commas:
+    max-spaces-before: 0
+    min-spaces-after: 1
+    max-spaces-after: 1
+  comments:
+    level: warning
+    require-starting-space: yes
+    min-spaces-from-content: 2
+  comments-indentation:
+    level: warning
+  document-end: disable
+  document-start:
+    present: yes
+  empty-lines:
+    max: 2
+    max-start: 0
+    max-end: 0
+  hyphens:
+    max-spaces-after: 1
+  indentation:
+    spaces: consistent
+    indent-sequences: yes
+    check-multi-line-strings: no
+  key-duplicates: enable
+  line-length:
+    max: 200
```

Comment:
200 seems like a lot.  I've seen a lot of modern projects go with 100, a few go with 120, and older projects shoot for 80.
100 is considered the length that allows for two columns of text on a larger monitor (i.e. `tput cols` is just over 200)
80 is considered the length that allows for two columns of text on a smaller/laptop monitor (i.e. `tput cols` is just over 160).  Obviously we don't need to maintain compatibility with old TTY's.

My vote is that we try 80.  If we run the linter and realize need to bump it to 100, we can do that.  If we need to go over 100, maybe we should rewrite that section entirely.

Reply:
I like 120 :+1:

Reply:
My preference is 100